src/main/java/com/launchacademy/reviews/models/ReviewForm.java

```diff
@@ -16,7 +16,7 @@ public class ReviewForm {
   @NotBlank
   private String name;
 
-  @NotBlank
+  @NotNull
```

Comment:
Nice catch! Be sure to always test happy and sad paths when putting in a new feature.  For me writing down some of the edge cases and checking them off as I go really helps